om4labs/diags/heat_transport/heat_transport.py

```diff
@@ -46,7 +46,8 @@ def compute(advective, diffusive=None, vmask=None, rho0=1.035e3, Cp=3989.0):
         HT = advective + diffusive
     else:
         HT = advective
-    HT = HT.mean(dim="time")
+    if len(HT.time) > 1:
```

Comment:
I would just add a warning to say that non-weighted time-average is performed

Reply:
Good idea - I agree.